### ISSUE TITLE
🔴 Blog: Default link colours don't pass accessibility tests

### ISSUE THREAD

**Issue by helloCaller** [jakemoves/cohort-server#246]
The soft blue default link colour on a white background only passes the Colour Contrast Analyser accessibility tests for large text. Fails on all other text sizes.

I'd suggest: #0503FF or something in and around there.
